packages/uiweb/src/lib/components/space/SpaceWidget/LiveWidgetContent.tsx

```diff
@@ -58,12 +96,25 @@ export const LiveWidgetContent: React.FC<LiveWidgetContentProps> = ({
 
   const isMicOn = spaceObjectData?.connectionData?.local?.audio;
 
-  const numberOfRequests = spaceObjectData.liveSpaceData.listeners.filter((listener: any) => listener.handRaised).length;
+  const numberOfRequests = spaceObjectData.liveSpaceData.listeners.filter(
+    (listener: any) => listener.handRaised
+  ).length;
 
   const handleMicState = async () => {
     await spacesObjectRef?.current?.enableAudio?.({ state: !isMicOn });
   };
 
+  const handleShowEmojiContainer = () => {
+    setShowEmojiContainer((prev) => !prev);
+  };
```

Comment:
I should be able to click outside anywhere on the space container or elsewhere on the screen and the emojji selector popup should close. Implement this using useRef. Check `dropdownRef` in ProfileContainer.tsx for reference.